fe/src/constants/path.ts

```diff
@@ -1,12 +1,29 @@
 export const PATH = {
   home: '/',
+  newProduct: '/products/new',
   sales: '/sales',
   interests: '/interests',
   chat: '/chat',
-  auth: '/auth',
+  account: '/account',
   redirect: '/oauth/redirect',
-  signup: '/auth/signup',
+  signup: '/signup',
   notFound: '/*',
+  detail: '/detail',
+};
+
```

Comment:
아래 END_POINT가 상수의 역할을 하지 못하고 너무 복잡해진 것 같아요. query나 path는 제외하고 심플하게 상수만 정의하는건 어떨까요?

Reply:
```suggestion
export const END_POINT = {
  locations: '/api/users/locations',
  nicknameCheck: `/api/users/nickname`,
  signup: `/api/users/signup`,
  login: `/api/users/login`,
  logout: `/api/users/logout`,
  refreshToken: `/api/users/reissue-access-token`,
  categories: `/api/categories`,
  products: '/api/products',
  imageUpload: `/api/images`,
};
```
이런 식으로 상수로 가져다 쓸 수 있게 정해두고 사용처에서 필요한 값들을 template literal로 합쳐서 사용해보라는 말씀이실까요?

Reply:
네 어차피 api는 동일하게 붙는다면 axios 초기화할때 설정해주고 나머지 path들을 상수로 가지고 있어도 될 것 같구요.
상수인데 메서드를 포함하고 있다보니 상수의 역할을 잘 하고 있다는 느낌은 안들어서요!